main.go

```diff
@@ -39,7 +40,24 @@ func main() {
 		log.Fatal(err)
 	}
 
-	logger := server.NewLogger(cfg)
+	// Initialize the logger - begin
+	logger, err := mlog.NewLogger()
+	if err != nil {
+		log.Fatal(err)
+	}
+	cfgJSON := cfg.LoggingCfgJSON
+	if cfg.LoggingCfgFile == "" && cfgJSON == "" {
+		// if no logging defined, use default config (console output)
+		cfgJSON = server.DefaultLoggingConfig(cfg)
+	}
+	err = logger.Configure(cfg.LoggingCfgFile, cfgJSON, nil)
+	if err != nil {
+		log.Fatal("Error in config file for logger: ", err)
+		return
+	}
+	defer func() { _ = logger.Shutdown() }()
+	// Initialize the logger - end
+
```

Comment:
I wonder if it is worth it (and doable) adding tests for this logic.

Reply:
What should the test do?

Reply:
I succeeded!